Scripts/Stage_Select/StageSelect.gd

```diff
@@ -1,16 +1,45 @@
 extends Node2D
 
-
+var levelCompletedTheme = load("res://Themes/LevelCompletedButton.tres")
+var levelUncompletedTheme = load("res://Themes/LevelUncompletedButton.tres")
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
-	pass # Replace with function body.
-
-
-# Called every frame. 'delta' is the elapsed time since the previous frame.
-#func _process(delta):
-#	pass
-
+	set_stages()
+	
+
+#Calls set_windows() on all stage buttons in the scene
+func set_stages():
+	var stages = get_children()
+	stages.erase($BackButton)
+	
+	for stage in stages:
+		var stageName = stage.name.replace("Button", "")
+		set_windows(stage, SceneSwapper.stageLevelCounts[stageName])
+	
+
+#Adds windows based on how many levels there are in the stage, yellow if level is completed, gray if not
+func set_windows(stage, levelCount):
+	var stageName = stage.name.replace("Button", "")
+	var windowSize = Vector2(75, 75)
+	var posx = 50
+	var posy = 15
+	
+	for i in range(1, levelCount + 1):
+		#Set position & size of window
+		var window = Panel.new()
+		window.set_position(Vector2(posx + (100 * (i - 1)), posy))
+		window.set_size(windowSize)
```

Comment:
Make sure that the panels will be drawn underneath the label node for each stage select. Rn I think they're drawn on top.